src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/metrics/classdataabstractioncoupling/ignore/deeper/Example4.java

```diff
@@ -0,0 +1,29 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="ClassDataAbstractionCoupling">
+      <property name="max" value="2"/>
+    </module>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.metrics.classdataabstractioncoupling.ignore.deeper;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicInteger;
+
+// xdoc section -- start
+// violation below, "Class Data Abstraction Coupling is 3 (max allowed is 2)."
+public class Example4 {
+  Set set = new HashSet(); // Ignored by default
+  Map map = new HashMap(); // Ignored by default
+  AtomicInteger atomicInteger = new AtomicInteger(); // Counted, 1
+  BigInteger bigInteger = new BigInteger("0"); // ...
```

Comment:
please do not make empty comments, we use comment as attention points

Reply:
I thought since it is for documentation, it would help emphasizing on the continuation of the counter without having to repeat. 
But understood